addons/point_of_sale/models/pos_payment_provider.py

```diff
@@ -0,0 +1,60 @@
+from odoo import api, fields, models
+
+
+class PosPaymentProvider(models.Model):
+    _name = 'pos.payment.provider'
+    _description = "Point of Sale Payment Provider (terminal)"
+    _order = 'module_state, state desc, name'
+
+    name = fields.Char(string='Name', required=True, translate=True)
+    code = fields.Selection([
+        ('none', 'No Provider Set')
+        ],
+        string='Code',
+        help='The technical code of this payment provider(terminal).',
+        default='none',
+        required=True,
+    )
+    state = fields.Selection([
```

Comment:
```suggestion
    mode = fields.Selection([
```

Opinion: I think `mode` is a more proper name for this.